include/zephyr/settings/settings.h

```diff
@@ -36,7 +36,7 @@ extern "C" {
 
 #define SETTINGS_MAX_DIR_DEPTH	8	/* max depth of settings tree */
 #define SETTINGS_MAX_NAME_LEN	(8 * SETTINGS_MAX_DIR_DEPTH)
-#define SETTINGS_MAX_VAL_LEN	256
+#define SETTINGS_MAX_VAL_LEN	512
```

Comment:
Why is this a noup? Shouldn't this be addressed upstream instead?

Reply:
@rado17  please raise a PR upstream

Reply:
I will drop this from this PR, we can raise a fix separately, don't want to hold this PR for this.